package.json

```diff
@@ -30,64 +30,56 @@
     "babel-preset-env": "^1.2.2",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-register": "^6.24.1",
     "babelify": "^7.3.0",
+    "browser-sync": "^2.18.8",
+    "browserify": "^14.3.0",
+    "browserify-incremental": "^3.1.1",
+    "del": "^2.2.2",
     "eslint": "3.18.0",
     "eslint-config-xo": "0.18.1",
     "eslint-plugin-security": "^1.3.0",
-    "grunt": "1.0.1",
-    "grunt-autoprefixer": "3.0.4",
-    "grunt-available-tasks": "0.6.3",
-    "grunt-banner": "0.6.0",
-    "grunt-bootlint": "0.10.2",
-    "grunt-browser-sync": "2.2.0",
-    "grunt-browserify": "^5.0.0",
-    "grunt-bump": "0.8.0",
-    "grunt-cache-bust": "^1.5.0",
-    "grunt-changelog": "0.3.2",
-    "grunt-contrib-clean": "1.0.0",
-    "grunt-contrib-connect": "1.0.2",
-    "grunt-contrib-copy": "1.0.0",
-    "grunt-contrib-cssmin": "2.0.0",
-    "grunt-contrib-htmlmin": "^2.3.0",
-    "grunt-contrib-imagemin": "1.0.1",
-    "grunt-contrib-less": "1.4.1",
-    "grunt-contrib-uglify": "2.2.0",
-    "grunt-contrib-watch": "1.0.0",
-    "grunt-eslint": "19.0.0",
-    "grunt-generator": "0.4.1",
-    "grunt-git": "1.0.3",
-    "grunt-githooks": "0.6.0",
-    "grunt-html": "8.3.2",
-    "grunt-jsdoc": "2.1.0",
-    "grunt-newer": "1.2.0",
-    "grunt-notify": "0.4.5",
-    "grunt-nsp": "^2.3.1",
-    "grunt-processhtml": "0.4.1",
-    "grunt-shell": "2.1.0",
-    "grunt-uncss": "0.6.1",
-    "load-grunt-tasks": "3.5.2",
-    "time-grunt": "1.4.0"
+    "gulp": "github:gulpjs/gulp#4.0",
+    "gulp-babel": "^6.1.2",
+    "gulp-changed": "^2.0.0",
+    "gulp-clean-css": "^3.0.4",
+    "gulp-concat": "^2.6.1",
+    "gulp-eslint": "^3.0.1",
+    "gulp-imagemin": "^3.2.0",
+    "gulp-less": "^3.3.0",
+    "gulp-notify": "^3.0.0",
+    "gulp-nsp": "^2.4.2",
+    "gulp-plumber": "^1.1.0",
+    "gulp-processhtml": "^1.1.0",
+    "gulp-rename": "^1.2.2",
+    "gulp-rollup": "^2.11.0",
+    "gulp-sourcemaps": "^2.6.0",
+    "gulp-uglify": "^2.1.2",
+    "husky": "^0.13.3",
+    "gulp-uncss": "^1.0.6",
+    "less-plugin-autoprefix": "^1.5.1",
+    "vinyl-buffer": "^1.0.0",
+    "vinyl-source-stream": "^1.1.0"
   },
   "scripts": {
-    "postinstall": "chmod u+x .postinstall.js && node ./.postinstall.js",
-    "test": "grunt lint",
-    "start": "grunt sync",
-    "dev": "grunt dev",
-    "serve": "grunt serve",
-    "dev:watch": "grunt",
-    "dev:sync": "grunt sync",
-    "watch": "grunt watch",
-    "docs": "grunt jsdoc",
-    "lint": "grunt lint",
-    "lint:fix": "grunt lint:fix",
-    "build": "grunt build",
-    "build:check": "grunt build:check",
-    "release:patch": "grunt release:patch",
-    "release:minor": "grunt release:minor",
-    "release:major": "grunt release:major"
+    "test": "",
+    "start": "gulp",
+    "dev": "gulp build",
+    "serve": "gulp serve",
+    "watch": "gulp watch",
+    "docs": "",
+    "lint": "gulp lint -prod",
+    "lint:fix": "gulp lint",
+    "build": "gulp build -prod",
+    "build:check": "gulp serve -prod",
+    "release:patch": "",
```

Comment:
see https://github.com/krnlde/knockout-undoredo/blob/master/gulpfile.babel.js#L102-L115 for reference